spec/card_generator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rspec"
+require "./lib/card_generator"
+
+RSpec.describe CardGenerator do 
+  describe "#initialize" do 
+    it "exists" do 
+      filename = "cards.txt"
+      card_generator = CardGenerator.new(filename)
+      expect(card_generator).to be_a CardGenerator
+    end
+
+    it "creates readable cards" do 
+      filename = "cards.txt" 
+      cards = CardGenerator.new(filename).cards
+      expect(cards.length).to eq 52
+      hearts = cards[0..12]
+      clubs = cards[13..25]
+      diamonds = cards[26..38]
+      spades = cards[39..51]
+      values = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"]
+      hearts.each_with_index do |card, i|
```

Comment:
This test is failing for me. Curious if it passes for you. For me, the cards are all off by one suit. Ex: The variable `hearts` are actually clubs.